dracut/module-setup.sh

```diff
@@ -68,25 +68,19 @@ install() {
     fi
   done
 
-  # Workaround for zfsonlinux/zfs#4749 by ensuring libgcc_s.so(.1) is included
-  _ret=0
-  # If zpool requires libgcc_s.so*, dracut will track and include it
-  if ! ldd "$( command -v zpool )" | grep -qF 'libgcc_s.so'; then
-    # On systems with gcc-config (Gentoo, Funtoo, etc.), use it to find libgcc_s
-    if command -v gcc-config >/dev/null 2>&1; then
-      dracut_install "/usr/lib/gcc/$(s=$(gcc-config -c); echo "${s%-*}/${s##*-}")/libgcc_s.so.1"
-      _ret=$?
-    # Otherwise, use dracut's library installation function to find the right one
-    elif ! inst_libdir_file "libgcc_s.so*"; then
-      # If all else fails, just try looking for some gcc arch directory
-      dracut_install /usr/lib/gcc/*/*/libgcc_s.so*
-      _ret=$?
+  # shellcheck disable=SC2154
+  if [ -z "${zfsbootmenu_skip_gcc_s}" ]; then
```

Comment:
This should probably be documented in zfsbootmenu.7 for both the dracut / initcpio options sections.

Reply:
I'm okay with leaving this undocumented until it becomes a real issue.

Reply:
I decided to document the behavior. Let me know if the wording needs adjustment.